index.js

```diff
@@ -23,61 +23,71 @@ class IntegrationAPI extends EventEmitter {
   #server;
   #clients;
 
-  constructor () {
+  constructor() {
     super();
 
-    this.#driverPath = 'driver.json';
+    this.#driverPath = "driver.json";
 
     // set default state to connected
     this.#state = uc.DEVICE_STATES.DISCONNECTED;
 
     this.#clients = new Map();
 
     // create storage for available and configured entities
-    this.availableEntities = new Entities('available');
-    this.configuredEntities = new Entities('configured');
+    this.availableEntities = new Entities("available");
+    this.configuredEntities = new Entities("configured");
 
     // connect to update events for entity attributes
-    this.configuredEntities.on(
-      uc.EVENTS.ENTITY_ATTRIBUTES_UPDATED,
-      async (entityId, entityType, attributes) => {
-        const data = {
-          entity_id: entityId,
-          entity_type: entityType,
-          attributes: Object.fromEntries(attributes)
-        };
-
-        await this.#broadcastEvent(uc.MSG_EVENTS.ENTITY_CHANGE, data, uc.EVENT_CATEGORY.ENTITY);
-      }
-    );
+    this.configuredEntities.on(uc.EVENTS.ENTITY_ATTRIBUTES_UPDATED, async (entityId, entityType, attributes) => {
+      const data = {
+        entity_id: entityId,
+        entity_type: entityType,
+        attributes: Object.fromEntries(attributes)
+      };
+
+      await this.#broadcastEvent(uc.MSG_EVENTS.ENTITY_CHANGE, data, uc.EVENT_CATEGORY.ENTITY);
+    });
   }
 
-  init (driverPath) {
-    // load driver information from driver.json
-    this.#driverPath = driverPath;
-    let raw;
+  init(driverConfig) {
```

Comment:
Could you please a JSDoc comment describing the type of the `driverConfig` parameter?
This helps a lot dealing with multi-type arguments and IDE validation.
Something like:
```
  /**
   * Initialize the library
   * @param {string|object} either a string to specify the driver configuration file path, or an object holding the configuration
   */
```